docs/recipes/authentication.md

```diff
@@ -40,8 +40,8 @@ export class AuthState {
   @Action(Login)
   login({ patchState }: StateContext<AuthStateModel>, { payload }: Login) {
     return this.authService.login(payload).pipe(tap((result: { token: string }) => {
-      patchState({ token, username: payload.username });
-    }))
+      patchState({ result.token, username: payload.username });
```

Comment:
This is a syntax error.
Should be `token: result.token`.

Reply:
Good catch ))

Reply:
When the fix not fix. hehe